plugins/chain_api_plugin/chain.swagger.yaml

```diff
@@ -89,6 +89,59 @@ paths:
             application/json:
               schema:
                 $ref: "https://eosio.github.io/schemata/v2.1/oas/BlockInfo.yaml"
+  /get_finalizer_info_result:
+    post:
+      description: Returns information about finalizers
+      operationId: get_finalizer_info_result
+        responses:
+          "200":
+            description: OK
+            content:
+              application/json:
+                schema:
+                  active_finalizer_policy:
+                    type: array
+                    description: Variant type, an array of strings with the current finalizers
```

Comment:
I think I would remove `Variant type, ` that is an implementation detail. The JSON is an object with `generation`, `threshold`, and an array of `finalizers`. Seems like that all should be described.
```
 "active_finalizer_policy": {
      "generation": 1,
      "threshold": 3,
      "finalizers": [
        {
          "description": "finalizer #1",
          "weight": 1,
          "public_key": "PUB_BLS_PcJG7lYgwZcsFGe-i4WWYDw1GtP7r5vGWFcRe97TVcp5f5_h6RRRPRhZSGKtRF4CtdJxAP50VewH4VIhMIPfhsu9C4GGH4MgOzSwl51N3EK6XiAiRHQ_lcGvxggipYwIReHkhQ"
        },
```